src/radio/__tests__/RadioGroup.test.js

```diff
@@ -49,5 +49,12 @@ describe('Radio Group', () => {
     expect(onChangeMock).toBeCalledTimes(1)
   })
 
+  it('should render the radio buttons with HTML name attributes specified', () => {
```

Comment:
Can we add a test to make sure each radio input still has a name if not passed in? (I realize there's no tests for this currently, but always good to backfill coverage when we're updating stuff anyway!)

Reply:
Sure. I will add a test for the case when the prop is not passed in

Reply:
I added new test for the case `name` property is empty @brandongregoryscott